ui/apps/platform/src/Containers/MainPage/Header/CLIDownloadMenu.tsx

```diff
@@ -16,95 +34,61 @@ type CLIDownloadMenuProps = {
-                })
                 .catch((err) => {
                     addToast(`Error while downloading roxctl for ${os}`);
                     removeToast();
                     Raven.captureException(err);
+                })
+                .finally(() => {
+                    removeToast();
                 });
         };
     }
 
-    const appLauncherItems = [
-        <ApplicationLauncherItem
-            key="app-launcher-item-cli-mac-amd64"
-            component="button"
-            onClick={handleDownloadCLI('darwin-amd64')}
-        >
-            Mac x86_64
-        </ApplicationLauncherItem>,
-        <ApplicationLauncherItem
-            key="app-launcher-item-cli-mac-arm64"
-            component="button"
-            onClick={handleDownloadCLI('darwin-arm64')}
-        >
-            Mac arm_64
-        </ApplicationLauncherItem>,
-        <ApplicationLauncherItem
-            key="app-launcher-item-cli-linux-amd64"
-            component="button"
-            onClick={handleDownloadCLI('linux-amd64')}
-        >
-            Linux x86_64
-        </ApplicationLauncherItem>,
-        <ApplicationLauncherItem
-            key="app-launcher-item-cli-linux-arm64"
-            component="button"
-            onClick={handleDownloadCLI('linux-arm64')}
-        >
-            Linux arm_64
-        </ApplicationLauncherItem>,
-        <ApplicationLauncherItem
-            key="app-launcher-item-cli-linux-ppc64le"
-            component="button"
-            onClick={handleDownloadCLI('linux-ppc64le')}
-        >
-            Linux ppc64le
-        </ApplicationLauncherItem>,
-        <ApplicationLauncherItem
-            key="app-launcher-item-cli-linux-s390x"
-            component="button"
-            onClick={handleDownloadCLI('linux-s390x')}
-        >
-            Linux s390x
-        </ApplicationLauncherItem>,
-        <ApplicationLauncherItem
-            key="app-launcher-item-cli-windows-amd64"
-            component="button"
-            onClick={handleDownloadCLI('windows-amd64')}
-        >
-            Windows x86_64
-        </ApplicationLauncherItem>,
-    ];
-
-    function toggleCLIMenu() {
-        setIsCLIMenuOpen(!isCLIMenuOpen);
-    }
-
-    const CLIDownloadIcon = (
-        <Flex alignItems={{ default: 'alignItemsCenter' }} spaceItems={{ default: 'spaceItemsSm' }}>
-            <FlexItem>
-                <DownloadIcon alt="" />
-            </FlexItem>
-            <FlexItem>CLI</FlexItem>
-        </Flex>
-    );
-
     return (
-        <ApplicationLauncher
-            aria-label="CLI Download Menu"
-            onToggle={toggleCLIMenu}
+        <Dropdown
             isOpen={isCLIMenuOpen}
-            items={appLauncherItems}
-            position="right"
-            data-quickstart-id="qs-masthead-climenu"
-            toggleIcon={CLIDownloadIcon}
-        />
+            onOpenChange={(isOpen) => setIsCLIMenuOpen(isOpen)}
+            onOpenChangeKeys={['Escape', 'Tab']}
```

Comment:
Today I learned. According links in reply to ai, I have a question.
* Before I read, I was going to ask about relevance for 14 existing occurrences of `Dropdown` element.
* After I read, did the default not provide keyboard accessibility?

> Keys that trigger onOpenChange, defaults to tab and escape. It is highly recommended to include Escape in the array, while Tab may be omitted if the menu contains non-menu items that are focusable.

Reply:
In this case, no, the default behavior does not close the dropdown on either `Escape` or `Tab` presses. It seems this is probably relevant to the other dropdown cases, although it is strange that this is claimed to be the default? 🤔 

> Keys that trigger onOpenChange, defaults to tab and escape.

Reply:
Correction - it looks like this requires both `onOpenChange` and `onOpenChangeKeys` to be functional. Without either, or without both, the Esc and Tab presses do not close an open menu.

Reply:
Aha, so another candidate lint rule?

For your info, here are defaults for PatternFly 5 and 6:
https://github.com/patternfly/patternfly-react/blob/5.0.x/packages/react-core/src/components/Dropdown/Dropdown.tsx
https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/src/components/Dropdown/Dropdown.tsx